clang/runtime/dpct-rt/include/dpl_extras/vector.h

```diff
@@ -327,57 +560,59 @@ class device_vector {
   void insert(iterator position, size_type n, const T &x) {
     if (position == end()) {
       resize(size() + n);
-      ::std::fill(oneapi::dpl::execution::make_device_policy(get_default_queue()),
-                end() - n, end(), x);
+      ::std::fill(
```

Comment:
How about use `uninitialized_fill` instead of `std::fill` and `resize`?

Reply:
Yes, this should be fixed.  Thanks.

Reply:
fixed, thanks.